### PR TITLE
Add roadmap to README

### PR DESCRIPTION
**Pull request by cinnamon-msft**
## Summary of the Pull Request
Updated README.md with roadmap and updated FONTLOG

 
## PR Checklist
* [ ] Closes #xxx
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Requires FONTLOG.txt to be updated
* [ ] Requires [/images/cascadia-code.png](/microsoft/cascadia-code/blob/master/images/cascadia-code.png) and/or [/images/cascadia-code-characters.png](/microsoft/cascadia-code/blob/master/images/cascadia-code-characters.png) to be updated
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Validation Steps Performed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,13 +65,22 @@ Please help us keep this repository clean, inclusive, and fun! We will not toler
...
 
-Michael Niksa, Senior Developer: [@michaelniksa](https://twitter.com/michaelniksa)
+This is the planned roadmap for Cascadia Code. Please be aware that the delivery dates are estimates and the features may arrive slightly earlier or later than predicted. This roadmap will continuously be updated as further features come along.
 
-Rich Turner, Program Manager: [@richturn_ms](https://twitter.com/richturn_ms)
+| Delivery Date | Feature | Description |
+| ------------- | ------- | ----------- |
+| October 2019 | Conversion of GitHub Pipeline/Workflow | Converting the GitHub pipeline to UFO to allow for users who don't have Macs to contribute to the source code. |
+| November 2019 | Character Set Extensions | The addition of Greek, Cyrillic, and Vietnamese, and other characters to the main character set. |
+| March 2020 | Weight Axis |Converting Cascadia Code into a variable font with milestone light and bold weights. |
```
Comment (DHowett-MSFT) [inline README.md]:
```suggestion
| March 2020 | Weight Axis | Converting Cascadia Code into a variable font with milestone light and bold weights. |
```

File: README.md [hunk 2]
```diff
@@ -65,15 +65,24 @@ Please help us keep this repository clean, inclusive, and fun! We will not toler
...
 
-Michael Niksa, Senior Developer: [@michaelniksa](https://twitter.com/michaelniksa)
+This is the planned roadmap for Cascadia Code. Please be aware that the delivery dates are estimates and the features may arrive slightly earlier or later than predicted. This roadmap will continuously be updated as further features come along.
 
-Rich Turner, Program Manager: [@richturn_ms](https://twitter.com/richturn_ms)
+| Delivery Date | Feature | Description |
+| ------------- | ------- | ----------- |
+| October 2019 | Conversion of GitHub Pipeline/Workflow | Converting the GitHub pipeline to UFO to allow for users who don't have Macs to contribute to the source code. |
+| November 2019 | Character Set Extensions | The addition of Greek, Cyrillic, and Vietnamese, and other characters to the main character set. |
+| March 2020 | Weight Axis | Converting Cascadia Code into a variable font with milestone light and bold weights. |
```
Comment (mdtauk) [inline README.md]: I assume these **Light** and **Bold** weights are in addition to a Regular/Normal/Medium weight? @cinnamon-msft

Comment (DHowett-MSFT) [reply README.md]: If it's going to have a variable weight axis, is it worth asking that question?

Comment (mdtauk) [reply README.md]: > If it's going to have a variable weight axis, is it worth asking that question?

Well yes of course, but the specificity in calling out Light and Bold as milestones, and the feedback saying the current font weight feels too heavy - I am just asking for clarification. 😀

**Pull request merged**